resources/assets/js/components/ApplicantProfile/ProfileBasicInformation.tsx

```diff
@@ -0,0 +1,219 @@
+      if ( (JSON.stringify(item) == JSON.stringify(jsonObj)) ) {
+        flag = 0
+        return
+      }
+      flag = 1
+    })
+
+    // If the flag equals one, then it is a new key value pair. Add the record to the array, as it is unique
+    if (flag == 1) {
+      uniqueClassifications.push(jsonObj)
+    }
+  })
+
+  function getLevelsOfClassification(classificationKey : string) {
+    let correspondingGocClassifications : GocClassification[] = [];
+    gocClassifications.forEach(function( item : GocClassification) {
+      if (item.classification.key == classificationKey) {
+        correspondingGocClassifications.push(item)
+      }
+    })
+
+    let correspondingLevels : Number[] = [];
+    correspondingGocClassifications.forEach(function(correspondingGocClassification : GocClassification) {
+      correspondingLevels.push(correspondingGocClassification.level)
+    })
+
+    return correspondingLevels
+  }
+
+  return (
+    <>
+      <div data-c-grid-item="base(1of3)" data-c-grid="gutter top">
+        <div data-c-grid-item="base(1of1) tl(1of2)" data-c-input="select">
+          <div>
+            <i className="fas fa-caret-down" />
+            <select  required id="SEL2" onChange={handleSelectedClassification} >
+              {uniqueClassifications.map(item =>
+                <option key={item.id} value={item.id}>{item.key}</option>
+              )};
+            </select>
+          </div>
+          <span>This input has an error.</span>
+        </div>
+        <div data-c-grid-item="base(1of1) tl(1of2)" data-c-input="select">
+          <div>
+            <i className="fas fa-caret-down" />
+            <select required id="SEL2">
+              {
+                getLevelsOfClassification(getSelectedClassification()).map(item => {
+                  <option>{item}</option>
+                })
+              }
+              {/*
+              <option>01</option>
+              <option>02</option>
+              <option>03</option>
+              <option>04</option>
+              <option>05</option>
+               */}
+            </select>
+          </div>
+          <span>This input has an error.</span>
+        </div>
+      </div>
+    </>
+  );
+};
+
+export const ProfileBasicInformation: React.FC<ProfileBasicInformationProps> = ({
+  gocClassifications
+}) => {
+  const intl = useIntl();
+
+  return (
+    <>
+      <div>
+        <h2 data-c-heading="h2" data-c-margin="bottom(1)">
+          {intl.formatMessage(myBasicInformationMessages.heading)}
+        </h2>
+        <p data-c-margin="bottom(1)">
+          <FormattedMessage
+            id="profile.experience.preamble"
+            defaultMessage="This profile is also shared when you submit a job application."
+            description="First section of text on the 'My Basic Information' of the Application Timeline."
+          />
+        </p>
+        <div>
+          <p>
+            Name: <b data-c-color="c1">Gerardi Escandon</b>{" "}
```

Comment:
These labels, and other english text, should also be localized. The Name should come from a ProfileBasicInformation object passed into the component.